cfme/networks/provider/nuage.py

```diff
@@ -77,7 +77,7 @@ def __attrs_post_init__(self):
     @property
     def mgmt(self):
         from cfme.utils.providers import get_mgmt
-        d = self.data
+        d = copy.deepcopy(self.data)
```

Comment:
**OPTIONAL** at this specific use-cases a shallow copy might be sufficient

Reply:
Good one, you're right. I would go with deep copy anyway because we pass it forward to `get_mgmt()` and god knows what kind of hanky-panky things it does to it. Unless you insist, then I'll make it shallow, of course.